traffic_ops/testing/api_contract/v4/pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+
+log_cli = 1
+log_cli_level = INFO
+log_cli_format = %(asctime)s [%(levelname)8s] %(message)s (%(filename)s:%(lineno)s)
+log_cli_date_format=%Y-%m-%d %H:%M:%S
```

Comment:
We should use RFC3339 for consistency throughout the project. That would be `%Y-%m-%dT%H:%M:%S`, optionally with a `%z` suffix.

Reply:
Used RFC3339 for date format.